languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -47315,6 +47315,32 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                 <example correction="la motte">C'est <marker>lamotte</marker> de terre.</example>
             </rule>
         </rulegroup>
```

Comment:
Speller is correcting _halene_ to _haleine_, but not _halène_... Probably since _halener_ is a verb, so "Le chien halène" is actually ok. 
![image](https://github.com/languagetool-org/languagetool/assets/114988314/5aaac500-2b46-4593-9bd1-d5944a8223be)

Reply:
Yes, the rulegroup focuses mainly on the confusion with the verb (I also added https://www.lalanguefrancaise.com/dictionnaire/definition/allene just in case), I made targeted patterns to avoid possible FPs, I think we're good :)